_posts/2017-11-14-Tiny-servant.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Tiny Servant"
+date:   2017-11-14
+source: Xanathar.169
+tags: [wizard, level3, transmutation]
+---
+
+**3rd-level Transmutation**
+
+**Casting Time**: 1 minute
+
+**Range**: Touch
+
+**Components**: V, S
+
+**Duration**: 8 hours
+
+You touch one Tiny, nonmagical object that isn't attached to another object or a surface and isn’t being carried by another creature. The target animates and sprouts little arms and legs, becoming a creature under your control until the spell ends or the creature drops to 0 hit points. See the stat block for its statistics.
```

Comment:
Add the stat block as a Markdown table.

Reply:
Where do you want me to put it? just at the end after the "higher levels" line?

Reply:
The best place would be right below this paragraph.